.gitignore

```diff
@@ -5,6 +5,7 @@
 /tmp
 /out-tsc
 /bin
+/data
```

Comment:
No such folder, please revert the changes

Reply:
done